Competition/src/main/cpp/Lift.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Lift.h"
+
+Lift::Lift() : liftMtrLeft{LiftConstants::VICTOR_ID_LEFT_Lift}, 
+             liftMtrRight{LiftConstants::VICTOR_ID_RIGHT_Lift},
+             liftServoLeft{LiftConstants::SERVO_ID_LEFT_Lift},
+             liftServoRight{LiftConstants::SERVO_ID_RIGHT_Lift},
+             operatorController(NULL) {
+    initLift();
+}
+
+void Lift::setController(frc::XboxController* controller) {
+    operatorController = controller;
+}
+
+void Lift::initLift() {
```

Comment:
rename to `init`. The ValorSubsystem that this implements has an init method that you need to override